prow/scripts/cluster-integration/helpers/reconciler.sh

```diff
@@ -143,6 +143,7 @@ function reconciler::initialize_test_pod() {
 
   # shellcheck disable=SC2086
   kc="$(cat ${KUBECONFIG})"
+  # Replace kyma version with latest release
```

Comment:
This function provides general usage to upgrade kyma by `KYMA_UPGRADE_SOURCE`, it can be `main` or a specific version if the value is not provided, so don't need to add this comment.